src/main/java/com/pokegoapi/api/map/pokemon/CatchablePokemon.java

```diff
@@ -457,7 +458,7 @@ protected CatchItemResult handle(ByteString result) throws RemoteServerException
 	 * @throws RemoteServerException if the server failed to respond
 	 */
 	public CatchItemResult useItem(ItemId item) throws LoginFailedException, RemoteServerException {
-		return useItemAsync(item).toBlocking();
+		return AsyncHelper.toBlocking(useItemAsync(item));
```

Comment:
looks like a transformer can´t change the type of a observable to this isn´t possible right now.